module/zstd/lib/zstd.c

```diff
@@ -8163,6 +8163,7 @@ const char* ZSTD_versionString(void) { return ZSTD_VERSION_STRING; }
 *  ZSTD Error Management
 ******************************************/
 #undef ZSTD_isError   /* defined within zstd_internal.h */
+#define ZSTD_isError zfs_ZSTD_isError
```

Comment:
@BrainSlayer Can't this be moved to the Compat_Wrapper or done without altering the sourcecode somehow?

Reply:
I've removed this for now, I think solving this issue is too far removed from updating ZSTD itself.
(as it's also an issue in 1.4.7 and is significantly more work.)